test/e2e/storage/volume_metrics.go

```diff
@@ -195,7 +195,7 @@ var _ = utils.SIGDescribe(framework.WithSerial(), "Volume metrics", func() {
 		framework.ExpectNoError(err, "failed to create Pod %s/%s", pod.Namespace, pod.Name)
 
 		err = e2epod.WaitTimeoutForPodRunningInNamespace(ctx, c, pod.Name, pod.Namespace, f.Timeouts.PodStart)
-		framework.ExpectError(err)
+		gomega.Expect(err).To(gomega.MatchError(framework.ErrFailure))
```

Comment:
`e2epod.WaitTimeoutForPodRunningInNamespace` + `expect error` -> this should be a `gomega.Consistently`.

Checking for `framework.ErrFailure` is too unspecific.